app/script/calling/CallingRepository.js

```diff
@@ -113,6 +113,7 @@ z.calling.CallingRepository = class CallingRepository {
    */
   shareCallStates() {
     this.mediaRepository.streamHandler.calls = this.calls;
+    this.mediaRepository.streamHandler.getCallById = this.getCallById.bind(this);
```

Comment:
We cannot let such an important part of the responsibilities of the `StreamHandler` be an optional property set.
Plus, in my opinion, the StreamHandler should not be the one calling `getCallById`.

Can the caller of the function `initiateMediaStream` be the also the one responsible for getting the call?
Why do we actually need the call in there, it doesn't seem to be used in the body of the function?
We need to find another solution for this.